samples/cpp/benchmark_app/README.md

```diff
@@ -197,76 +197,137 @@ This section provides step-by-step instructions on how to run the Benchmark Tool
    pip install openvino-dev
```

Comment:
Does the example work without installing a framework (e.g. openvino_dev[caffe])?

Reply:
Oops, when I changed the example model from `asl-recognition-0004` to `googlenet-v1`, I didn't test whether it would work without installing extra frameworks. It looks like `googlenet-v1` does require `[tensorflow2]` to be installed.  Thanks for catching that!

Per our other discussion, I am going to change the model back to `asl-recognition-0004`, which works with just `pip install openvino-dev`.